ysoserial/Plugins/MachineKeySessionSecurityTokenHandlerPlugin.cs

```diff
@@ -0,0 +1,146 @@
+﻿using System.Collections.Generic;
+using NDesk.Options;
+using System;
+using ysoserial.Generators;
+using System.IdentityModel;
+using System.IO;
+using System.Xml;
+using System.IdentityModel.Tokens;
+using ysoserial.Helpers;
+using System.IdentityModel.Services.Tokens;
+using AspNetTicketBridge;
```

Comment:
Instead of using `AspNetTicketBridge` which requires lots of .NET Core dependencies, use https://raw.githubusercontent.com/irsdl/ysoserial.net/refs/heads/SessionSecurityTokenHandler_Update/ysoserial/Helpers/MachineKeyHelper.cs and add it to your project and use it.  All imported dependencies should be removed as well (perhaps you want to apply all changes on a clean ysoserial.net code as there are many imports).